config/initializers/fast_gettext.rb

```diff
@@ -6,4 +6,4 @@
 FastGettext.text_domain = FastGettext.default_text_domain = 'rmt'
 
 # enable comments in .po/.pot files prefixed with i18n tag (see rxgettext -h)
-Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=i18n]
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=i18n] unless Rails.application.nil?
```

Comment:
`if Rails.application`.

I'd still move it into a separate initializer. Or better yet not use Rails initializer in CLI script (it's rather confusing that it's being used there -- why wouldn't `Rails.application` be defined in Rails initializer :smile:).

Reply:
I think the problem is that this initializer is used for both rails and `rmt-cli`. I guess we would need a new initializers directory to help prevent this happening again.

Reply:
Just put FastGettext stuff into `lib`, and then require it from Rails/CLI?

Reply:
I will try that